strawberry/field.py

```diff
@@ -102,6 +104,15 @@ def __call__(self, resolver: _RESOLVER_TYPE) -> "StrawberryField":
         if not isinstance(resolver, StrawberryResolver):
             resolver = StrawberryResolver(resolver)
 
+        for argument in resolver.arguments:
+            if isinstance(argument.type_annotation.annotation, str):
```

Comment:
We'll need to fix this in the future so that we can support forward references.

Reply:
i keep it in my mind
i will try to fix this later as you suggested